Geometry_Engine/Query/IsClockwise.cs

```diff
@@ -75,8 +75,14 @@ public static bool IsClockwise(this PolyCurve curve, Vector normal, double toler
                     cPts.Add(c.IEndPoint());
                 else if (c is Arc)
                 {
-                    cPts.Add(c.IPointAtParameter(0.5));
-                    cPts.Add(c.IEndPoint());
+                    cPts.Add((c as Arc).PointAtParameter(0.5));
+                    cPts.Add((c as Arc).EndPoint());
+                }
+                else if (c is Circle)
+                {
+                    cPts.Add((c as Circle).PointAtParameter(0.3));
```

Comment:
Maybe a detail, but why not use 1.0/3 and 2.0/3?

Reply:
Reducing number of calculations? I don't know, it can be changed but I don't think it will affect the results.

Reply:
I think the extra calcs are nothing compared to the whole computation. Main reason for using fractions is readability.